packages/pwa-kit-dev/scripts/upload.js

```diff
@@ -13,7 +13,6 @@ const DEFAULT_ORIGIN = process.env.CLOUD_API_BASE || 'https://cloud.mobify.com'
 
 const OPTION_DEFAULTS = {
     buildDirectory: 'build',
-    settingsFile: Utils.getSettingsPath(),
```

Comment:
It's probably best to leave the default for `credentialsFile` in the default options and not rely solely on it being passed in as an option. Although we aren't going to run into an issue as the code sits, down the road it might be expected that there is a default value.

Reply:
Added back.